packages/playground-ui/src/ds/components/Dialog/dialog.css

```
@@ -0,0 +1,52 @@
+  }
+}
+@keyframes dialog-content-out {
+  from {
+    opacity: 1;
+    transform: scale(1);
+  }
+  to {
+    opacity: 0;
+    transform: scale(1.02);
+  }
+}
```

Stylelint (17.9.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)